p2p/lightchain.py

```diff
@@ -257,14 +260,14 @@ def register_peer(self, peer: BasePeer) -> None:
             # TODO push validation into process pool executor
             VM.validate_header(header, parent_header)
 
-    async def _cleanup(self):
+    async def _cleanup(self) -> None:
         self.logger.info("Stopping LightPeerChain...")
 
-    async def _wait_for_reply(self, request_id: int) -> Dict[str, Any]:
+    async def _wait_for_reply(self, request_id: int) -> Any:
```

Comment:
Why this change?

Reply:
Good point, fixed and explained here https://github.com/ethereum/py-evm/pull/947/commits/6cf408c432c74f78d2029ced398774a6c85f142d